### PR TITLE
versions: Upgrade to k8s 1.15

### PR DESCRIPTION
**Pull request by jcvenegas**
Bump CI to kubernetes 1.15.

Fixes: #1860

Signed-off-by: Jose Carlos Venegas Munoz <jose.carlos.venegas.munoz@intel.com>

### CODE REVIEW

**Comment by grahamwhaley** [comment]
Some CIs failed @jcvenegas . From 18.04 for instance..

```
22:05:02 HEAD is now at 485227d72 Version 1.15.0
22:05:02 /tmp/jenkins/workspace/kata-containers-runtime-ubuntu-18-04-PR/go/src/github.com/kata-containers/tests
22:05:02 /tmp/jenkins/workspace/kata-containers-runtime-ubuntu-18-04-PR/go/src/github.com/cri-o/cri-o/test /tmp/jenkins/workspace/kata-containers-runtime-ubuntu-18-04-PR/go/src/github.com/kata-containers/tests
22:05:02 Ensure crio service is stopped before running the tests
22:05:02 Ensure docker service is stopped before running the tests
22:05:11 ++ time bats --jobs 16 --tap ctr.bats
22:05:11 1..45
22:05:22 not ok 1 ctr not found correct error message
22:05:22 # (from function `setup_crio' in file helpers.bash, line 258,
22:05:22 #  from function `start_crio' in file helpers.bash, line 308,
22:05:22 #  in test file ctr.bats, line 10)
22:05:22 #   `start_crio' failed
22:05:22 # time="2019-07-05T21:05:13Z" level=error msg="File descriptor 3 (/tmp/parXQCSO.par (deleted)) leaked on pvdisplay invocation. Parent PID 73960: /tmp/jenkins/workspace/kata-con\nFile descriptor 6 (/dev/mapper/control) leaked on pvdisplay invocation. Parent PID 73960: /tmp/jenkins/workspace/kata-con\n  Failed to find physical volume \"/dev/sdb\".\n" error="exit status 5"
22:05:22 # time="2019-07-05T21:05:18Z" level=error msg="error importing dir:/tmp/jenkins/workspace/kata-containers-runtime-ubuntu-18-04-PR/go/src/github.com/cri-o/cri-o/.artifacts/redis-image: Error committing the finished image: error adding layer with blob \"sha256:ff3a5c916c92643ff77519ffa742d3ec61b7f591b6b7504599d95a4a41134e28\": error creating layer with ID \"cd7100a72410606589a54b932cabd804a17f9ae5b42a1882bd56d263e02b6215\": devicemapper: Non existing device storage-thinpool"
22:05:22 # time="2019-07-05T21:05:20Z" level=fatal msg="failed to connect: failed to connect: context deadline exceeded"
22:05:22 # time="2019-07-05T21:05:22Z" level=fatal msg="failed to connect: failed to connect: context deadline exceeded"
```

**Comment by amshinde** [comment]
@jcvenegas Can you resolve the conflicts here

File: versions.yaml [hunk 1]
```diff
@@ -188,7 +188,7 @@ externals:
     description: |
       OCI-based Kubernetes Container Runtime Interface implementation
     url: "https://github.com/cri-o/cri-o"
-    version: "v1.14.6"
+    version: "v1.15.1"
```
Comment (GabyCT) [inline versions.yaml]: This needs to be replaced to 1.15.0

**Comment by GabyCT** [comment]
@jcvenegas tested locally the `kubernetes` tests with these changes, including the comment that I said and is working fine

**Comment by jodh-intel** [comment]
Any update on this @jcvenegas?

**Pull request merged**